modules/tinymce/CHANGELOG.md

```diff
@@ -4,15 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
-
-### Fixed
-- Setting the content with an attribute that contains a self-closing HTML tag did not preserve the tag. #TINY-10088
-- Resize handles would not appear on editable images in a non-editable context. #TINY-10118
-
-### Improved
-- Improved the tooltips of picker buttons for the urlinput components in the "Insert/Edit Image", "Insert/Edit Link" and "Insert/Edit Media" dialogs. #TINY-10155
-
 ## 6.7.0 - 2023-08-30
```

Comment:
Why is it both here and in the `.changes` folder? 🤔  If we want to keep this file we should keep all changes in it. If we don't, it should be removed and git ignored.

Reply:
I think we need to have the changelog.md file that gets generated commited in the tinymce repo so that we can point to that and review it easily. I think the different .md files in the .changes need to exist and be commited to so that you can merge that with new changes based on the previous version. There will be a lot of files but it would be more clear on what we have here. But I wonder if we should split apart all of the changelog into separate files here even though we wouldn't use it like that I think the idea is that it would combine all the different versions into one changelog. Having just one file containing all the changes like it is now if kind of not how it supposed to work but I guess there is nothing really blocking us from using it that way.

Reply:
We can't commit the generated file or we'll continue to have conflicts 😞 

I agree we need a way to review it and compare, but it will need to be in local (or CI) compiled output.

Splitting the old changelog into individual files could be interesting, but it sounds like a fair chunk of work to script up and I'm not sure what the benefit would be.

Reply:
No it would only be generated when you release so there would not be any conflicts then. It's all just separate files until you do that step.
Regarding the splitting it just a bit weird to have a 6.7.0 release having all the data. Going forward it will not be like that for future things then every file would be just that release data but I'm fine with it being weird if everyone understands why it's weird. But you are right it doesn't bring us any value beyond that to split it.

Reply:
the `CHANGELOG.md` when finished should only have released log, unreleased will be kept inside `.changes/unreleased`, each entry will have a separate file, conflict should not be a concern.

The only issue right now, is when running `changie batch/merge`, all the unreleased changes will be merged into the changelog, from what I see there is no config that can be used to select which change should be added, the only way to do so is manually add the changes into a new change log then run `changie merge`

Reply:
That still doesn't explain why there is a duplicate of the release changelog in `.changes`. Is that just how changie works? I see it's been added for every other project as well.

Reply:
Yes, that is how changie works it basically just concats things. So building a complete change log is to concat all the separate version changelogs together. In this case however we haven't separated the changelogs into those different files so 6.7.0 contains all the history not only the changes that where in the 6.7.0 release so that is why I suggested splitting it since then it would concat like it's supposed to work and make more sense.

Reply:
Right, I see now in the changie quick start guide it says to do this. OK, makes sense.